amethyst_rendy/src/bundle.rs

```diff
@@ -247,13 +235,14 @@ pub struct TargetMetadata {
     layers: u16,
 }
 
-#[derive(Debug)]
+#[derive(Debug, MutGetters)]
 struct PlanContext<B: Backend> {
     targets: HashMap<Target, TargetPlan<B>>,
     target_metadata: HashMap<Target, TargetMetadata>,
     passes: HashMap<Target, EvaluationState>,
     outputs: HashMap<TargetImage, ImageId>,
-    graph_builder: GraphBuilder<B, World>,
+    #[get_mut = "pub"]
```

Comment:
how about keeping the name `graph` instead of `graph_mut` here? Getting the graph builder for readonly access doesn't make much sense in practice. I'd rather keep the `PlanContext` API consistent with `TargetPlanContext`. It is currently not public, but very likely will be very soon.

Reply:
Would it make more sense to change `TargetPlanContext::graph()` to `TargetPlanContext::graph_mut()` then? getset doesn't support custom names (aside from changing `graph_mut` to `mut_graph`) so `graph` would have to be written by hand again to keep the name `graph` while returning a mutable reference.

Reply:
Although getting something immutably may not be useful, I think it'd be best to preserve convention and make all accessors uniform